dev/generate-css-json.js

```diff
@@ -86,11 +86,11 @@ function removeProperty(styles, property, removedProperties) {
 }
 
 /**
- * @param {import('css-style-applier').RawStyleData} rules
+ * Manually formats JSON for improved compactness.
```

Comment:
The rationale behind `RawStyleData` is not necessarily "for improved compactness", but rather easier parseability. JSON is obviously native in Javascript, but CSS parsing would be more cumbersome to perform, so the real intent was to avoid the need to either 1) create/import a CSS parser or 2) use some ugly workaround that is potentially not supported in all browser contexts for parsing the CSS.